cli.js

```diff
@@ -26,7 +26,7 @@ const cli = meow(`
       USD: 261.91
 `);
 
-const input = cli.input.map(item => item.replace(/,|\s+/g, '')).join(',');
+const input = cli.input.map(item => item.replace(/,\s+/g, '')).join(',');
```

Comment:
Looks like the same is logic is happening in the API(LOC 7), you should be able to just turn the array to a string and pass it to the API to format.